src/util/postgres_client.js

```diff
@@ -451,21 +451,22 @@ class PostgresTable {
         return {};
     }
 
-    async insertMany(data, options) {
-
-        let queries = [];
-        const ordered = _.isUndefined(options.ordered) ? true : options.ordered;
+    // This is done mainly to be quick
+    // Notice that the behaviour between MongoDB and PostgreSQL differs
+    // In PostgreSQL we either push everything at once or do not push anything at all
+    // In MongoDB we might push partially (succeed pushing several documents and fail on others), and it is done in parallel
+    async insertManyUnordered(data) {
+
+        const args = [];
+        let values_str = '';
```

Comment:
you can get `args` and `values_str` without the for loop:

```suggestion
        const args = _.flatten(data.map(doc => [doc._id, encode_json(doc)]));
        let values_str = _.times(data.length, i => `($${i*2+1} , $${i*2+2} )`).join(', ');
```

Reply:
your way seems to work too, but it was a bit hard to follow and read